tests/e2e/e2e_egress_test.go

```diff
@@ -17,7 +17,7 @@ var _ = OSMDescribe("HTTP and HTTPS Egress",
 		Bucket: 2,
 	},
 	func() {
-		Context("Egress", func() {
+		Context("CrossPlat Egress", func() {
```

Comment:
Is this renaming just temporary? Or are we going to use this naming convention going forward for all tests that should run on both Linux and Windows?

We can use this for now but we may want to reconsider when we start onboarding more tests and some are cross-plat, some just for linux, and some just for windows.

Reply:
Instead of naming it this way, would you mind extending the `OSMDescribeInfo struct to include the OS that be optionally set by tests, and use `-ginkgo.focus` to filter tests you want to run for Windows.